tmt/frameworks/beakerlib.py

```diff
@@ -66,7 +82,7 @@ def extract_results(
         beakerlib_results_filepath = invocation.path / 'TestResults'
 
         try:
-            results = invocation.phase.read(beakerlib_results_filepath, level=3)
+            bkr_results = invocation.phase.read(beakerlib_results_filepath, level=3)
```

Comment:
I'd recommend using `beakerlib_` prefix, if one is needed, `bkr_` does smell like "Beaker" which is something completely different.